### ISSUE TITLE
Add emoji reactions to posts, comments.

### ISSUE THREAD

**Issue by dessalines** [LemmyNet/lemmy-ui#29]

**Comment by n1trux** [comment]
It's two years later, do you still have a lot of issues that are more important?

Lemmy is similar to a lot of other platforms in some ways, e.g. that there's always a "starter post" with a title and no loosely connected graph of comments or posts. This is similar to both traditional forums (Flarum and Discourse have emoji reactions as addons) and Facebook (without the title, but Facebook has a singular top comment – and Emoji reactions). It is also similar to Matrix in that Lemmy has communities (which are like rooms, or rather… Matrix communities, which consist of a list of rooms, basically).

And it's also similar to messaging applications in that you can send messages to other users in a kind of "open group chat" and react to them with emoji.

IMHO Emoji could also be used to influence the positioning of posts, so a post with lots of happy emoji and lots of upvotes at the same time might be positioned higher than a post with the same amount of upvotes, but more angry faces.

**Comment by dessalines** [comment]
Yes, we have many other issues that are really important. There are currently only two devs working on lemmy. 

> IMHO Emoji could also be used to influence the positioning of posts, so a post with lots of happy emoji and lots of upvotes at the same time might be positioned higher than a post with the same amount of upvotes, but more angry faces.

I'm very much against this; lemmy already has votes for ranking / sorting, and picture emojis aren't comparable in any way.

**Comment by dullbananas** [comment]
If this is implemented, then it should be an opt-in feature that lets moderators add reaction types to a community

**Comment by L3v3L** [comment]
This might get super confusing for normal users. People might think adding a heart to a post is better than adding a upvote for example. Then you have to do the hard task of educating the users that emotes don't have any play in ranking. Fun idea though.

**Comment by alectrocute** [comment]
Not a fan of this one.
